docs/synapse/adminguides/usersroles.rstorm

```diff
@@ -0,0 +1,268 @@
+.. highlight:: none
+
+.. storm-cortex:: default
+
+.. _admin_create_users_roles:
+
+Create and Manage Users and Roles
+#################################
+
+A :ref:`gloss-user` account is required to authenticate to and access Synapse. Having "a Synapse account"
+effectively means having an account in the Cortex.
+
+In Synapse, a :ref:`gloss-role` can be used to "group" users with similar responsibilities (and related
+permissions requirements). You can **grant** or **revoke** one or more roles from a user.
+
+You grant (or deny) **permissions** to users or roles by assigning **rules** that specify those permissions
+(see :ref:`admin_perms`).
+
+Synapse includes the following built-in users and roles:
+
+- **Root** user. The **root** account has :ref:`admin_bkd_admin` privileges in the Cortex. The **admin**
+  status of the root account cannot be revoked, and the account cannot be locked / disabled.
+- **All** role. The **all** role has **read** access to the Cortex (specifically, to any view with
+  ``worldreadable=true``, which includes the **default** view). All user accounts are automatically granted
+  the **all** role (are part of the **all** "group"); this role cannot be revoked.
+
+.. TIP::
+  
+  The set of Storm :ref:`storm-auth` commands are collectively used to manage users, roles, and permissions
+  from Storm.
+  
+  In the commercial Optic UI, users, roles, and permissions can be managed through the **Admin Tool** and
+  through dialogs associated with various objects (such as Views or Stories), in addition to the Storm
+  commands.
+
+
+.. NOTE::
+  
+  The descriptions and examples below assume that you have deployed Synapse using native Synapse management
+  and authentication of users, roles, and permissions.
+  
+  The :ref:`devopsguide` includes information on provisioning **initial** users when Synapse is first deployed
+  (see :ref:`devops-task-users`). This guide focuses on ongoing management of users and roles once Synapse
+  is in production.
```

Comment:
"in production" vs "has access to storm" ?

`production` can have specific meanings for admins and devops people; whereas noting that they can use these tools once they have storm access is more specific.

Reply:
revised